### ISSUE TITLE
PUSHD usage is questionable

### ISSUE THREAD

**Issue by CAD97** [sagiegurari/cargo-make#87]
https://github.com/sagiegurari/cargo-make/blob/441d80e7062c4570fb5d8f9451159666e56b5647/src/runner.rs#L195-L201

Given an example virtual workspace `Cargo.toml` that I'm working with:

```toml
[workspace]
members = [
    "./mark-sweep/",
]
```

this generates the script line

```cmd
>PUSHD ./mark-sweep/
```

which is an invalid path to give [`pushd`](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/pushd) on Windows -- it only handles Windows paths where `\\` is the path separator instead of `//`.

For the correct behavior, the path string needs to be interpreted into a `Path` and then converted to the correct system-specific path representation.

[[[ ADDENDUM: Windows `cd`/`chdir` and `pushd` handle spaces in the path correctly, but UNIX `cd` doesn't: the path needs to be quoted (and is allowed to be on Windows) if it contains spaces. ]]]

**Comment by sagiegurari** [comment]
You are right. In your case however, I would expect you to simple define it as:

```toml
[workspace]
members = [
    "mark-sweep"
]
```

But anyhow, I'll fix it.

**Comment by sagiegurari** [comment]
A fix for this has been pushed to 0.10.8 branch. Once i'm finished with this release features, I'll publish a formal release.

**Comment by sagiegurari** [comment]
0.10.8 was just published with this fix.
Please verify it and I'll close this issue afterwards.
